xbmc/LangInfo.cpp

```diff
@@ -300,6 +302,7 @@ void CLangInfo::CRegion::SetGlobalLocale()
     strLocale = lcl.name();
     current_locale = current_locale.combine< std::collate<wchar_t> >(lcl);
     current_locale = current_locale.combine< std::ctype<wchar_t> >(lcl);
+    current_locale = current_locale.combine< std::time<wchar_t> >(lcl);
```

Comment:
They seem to be your only options. On which platform did you test this? Looking at the C++ documentation `std::time` doesn't have a template implementation so this is never going to work.